hera_cal/tests/test_io.py

```diff
@@ -707,13 +706,21 @@ def test_write_cal(self):
         if os.path.exists('ex.calfits'):
             os.remove('ex.calfits')
         # test single integration write
-        gains = odict(map(lambda k: (k, gains[k][:1]), gains.keys()))
-        uvc = io.write_cal("ex.calfits", gains, freqs, times[:1], return_uvc=True, outdir='./')
+        gains2 = odict(map(lambda k: (k, gains[k][:1]), gains.keys()))
+        uvc = io.write_cal("ex.calfits", gains2, freqs, times[:1], return_uvc=True, outdir='./')
         self.assertAlmostEqual(uvc.integration_time, 0.0)
         self.assertEqual(uvc.Ntimes, 1)
         self.assertTrue(os.path.exists('ex.calfits'))
         os.remove('ex.calfits')
 
+        # test multiple pol
+        for k in gains.keys():
+            gains[(k[0], 'y')] = gains[k].conj()
```

Comment:
Shouldn't this be 'Jyy'?

Reply:
In the test it was (1, 'x') so I just mirrored it to y. I don't think it matters in the context of this test, but yes in principle one could say it should it Jyy